packages/data/src/redux-store/test/index.js

```diff
@@ -286,3 +286,75 @@ describe( 'resolveSelect', () => {
 		] );
 	} );
 } );
+
+describe( 'normalizing args', () => {
+	it( 'should call the normalizeArgs method of the selector for both the selector and the resolver', async () => {
+		const registry = createRegistry();
+		const selector = () => {};
+
+		const normalizingFunction = jest.fn( ( ...args ) => args );
+
+		selector.normalizeArgs = normalizingFunction;
+
+		registry.registerStore( 'store', {
+			reducer: () => {},
+			selectors: {
+				getItems: selector,
+			},
+			resolvers: {
+				getItems: () => 'items',
+			},
+		} );
+		registry.select( 'store' ).getItems( 'foo', 'bar' );
+
+		expect( normalizingFunction ).toHaveBeenCalledWith( [ 'foo', 'bar' ] );
+
+		// Needs to be called twice:
+		// 1. When the selector is called.
+		// 2. When the resolver is fullfilled.
+		expect( normalizingFunction ).toHaveBeenCalledTimes( 2 );
+	} );
+
+	it( 'should not call the normalizeArgs method if there are no arguments passed to the selector (and thus the resolver)', async () => {
```

Comment:
@jsnajdr I reinstated the `arg?.length` check before calling the `normalizeArgs` method because if I didn't then this test will fail.